### ISSUE TITLE
What is the status of this project? Is it being actively maintained?

### ISSUE THREAD

**Issue by robert-gruner** [atlassian/better-ajv-errors#95]
Hi @torifat,

first of all a big thanks for this useful library. We've been using it in some business projects together with AJV and it was very handy for us.

I've seen that the last release was quite a while ago und that there are some pending PRs. So I wanted to check in on the state of the project.

- Can we assume that there is active maintenance for this lib?
- If you personally cannot continue, will there be any other Atlassian dev which carries on the support?
- Can we get "relatively short term" fixes to support AJV in v8?

Thanks in advance for your answer.

All the best,
Robert

**Comment by torifat** [comment]
Hey @robert-gruner ,

Sorry about the delayed reply. Sadly, I don't have enough capacity to actively maintain it at the moment. I added other maintainers before, but they moved on. I will see if I can find someone who is interested.

> I've seen that the last release was quite a while ago und that there are some pending PRs.

I started a new branch (https://github.com/torifat/better-ajv-errors/tree/typescript) and did a lot of work there -  https://github.com/atlassian/better-ajv-errors/projects/1. Because of that, I wasn't merging any new changes for some time. Then I hit a roadblock and was waiting for https://github.com/ajv-validator/ajv/issues/1090. It was finally fixed but I haven't had to chance to resume my work after that.

> Can we get "relatively short term" fixes to support AJV in v8?

I will see what I can do about the v8 support.

Cheers,
R
